google/cloud/security/scanner/scanner.py

```diff
@@ -133,6 +137,9 @@ def main(_):
 
     # If there are violations, send results.
     if all_violations:
+        for v in all_violations:
+            print _get_violation_project(v, snapshot_timestamp)
```

Comment:
I will remove it for now, since I can't find anything called _get_violation_project. We can add it later if needed